packages/salesforcedx-vscode-lwc/src/testSupport/testExplorer/iconPaths.ts

```diff
@@ -4,62 +4,51 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+
+import { extensionUris } from '@salesforce/salesforcedx-utils-vscode';
 import * as path from 'path';
+import * as vscode from 'vscode';
+import { VSCODE_LWC_EXTENSION_NAME } from '../../constants';
 import { TestResult, TestResultStatus } from '../types';
 
-const extensionPath = path.join(__filename, '..', '..', '..', '..', '..');
-const LIGHT_BLUE_BUTTON = path.join(
+const extensionPath = extensionUris.extensionUri(VSCODE_LWC_EXTENSION_NAME);
+const LIGHT_BLUE_BUTTON = extensionUris.join(
```

Comment:
The LWC constants were setup to be more easily updated so I went the easy path here instead of having two identical icon helpers.